clippy_lints/src/methods/mod.rs

```diff
@@ -107,6 +108,29 @@ declare_clippy_lint! {
     "used `cloned` where `copied` could be used instead"
 }
 
+declare_clippy_lint! {
+    /// ### What it does
+    /// Checks for usage of `_.cloned().last()`.
```

Comment:
The documentation should also reflect the complete lint.

Reply:
@llogiq I updated the documentation.